### ISSUE TITLE
C-z (suspend-frame) crashes EXWM (solution provided)

### ISSUE THREAD

**Issue by amirny2205** [emacs-exwm/exwm#36]
accidently pressing C-z or C-x C-z (bound to suspend-frame) crashes EXWM

solution:
(global-unset-key (kbd "C-z"))
(global-unset-key (kbd "C-x C-z"))

what would be the status for this thread "resolved but still accepts better answers"? probable org-mode...

**Comment by Stebalien** [comment]
Hm. We should probably advise that function and fail if it's an Emacs frame.

**Comment by minad** [comment]
I usually suggest to unbind/rebind C-z anyway (for GUI Emacs), since suspending is only useful when Emacs is used on the terminal. So maybe it is sufficient to document this problem and recommend a useful alternative command for C-z?

**Comment by Stebalien** [comment]
For suspend, yes. Although `suspend-frame` is still useful in GUI mode, just not for EXWM frames (it'll iconify GUI frames).

**Comment by minad** [comment]
Yes, we never iconify, so the recommendation in our case would be even
stronger. :) But probably there is nothing wrong with installing :override
advices for commands or functions which must never be used in EXWM. The
advice could simply print a message or throw an error. We could
introduce a variable similar to exwm-input--passthrough-functions.

**Comment by Stebalien** [comment]
> Yes, we never iconify, so the recommendation in our case would be even stronger.

Yep. My point is that it's not a blanket "don't do this in a GUI window".

> But probably there is nothing wrong with installing :override advices for commands or functions which must never be used in EXWM.

Or just have an `:arround` advice that checks if the frame is managed by EXWM (in theory, it would be nice to eventually support multiple X displays..., although I guess that kind of setup is basically dead at this point).
